app/creating-a-campaign/page.mdx

```diff
@@ -30,7 +30,7 @@ Here's a 2 minute video about how to create a campaign with Metrom.
 Select the DEX and pick the pool you want to incentivize.
 
 <div className="mediaWrapper">
-  ![Select DEX and pool](/images/select-dex-pool.png)
+  <img src="/images/select-dex-pool.png" alt="Select DEX and pool" />
```

Comment:
Why using `img =` tag directly? This way the image can't be opened with the zoom.